code/index.html

```diff
@@ -1,22 +1,79 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
-    <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
-</html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Project 2: The News Site</title>
+  <link rel="stylesheet" href="./style.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans+Condensed:wght@300&display=swap" rel="stylesheet">
+  <!-- dont forget the meta tag for responsiveness -->
+</head>
+
+<body>
+  <header>
+    <!-- Some kind of logo and navbar -->
+    <nav>
+      <div class=smiley>
+        <img class="logo" src="Smiley.png" alt=smiley>
+        <h1>The upBeat</h1>
+      </div>
+      <div class="hamburger">
+        <a href="/html/">Start</a>
+        <a href="/css/">Today</a>
+        <a href="/js/">Random</a>
+        <a href="/python/">Contact</a>
```

Comment:
the href paths seems a bit off here, right? html, css, js, python.

Reply:
I just used a copy from the internet. As my intention wasn't to use the links I left them, do you still think I should change it?

Reply:
yes, at least make them empty or go to the website it self instead of something out of the air.